File: back/src/main/java/com/example/back/controllers/dto/AddMatchDto.java
package com.example.back.controllers.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class AddMatchDto {
    private String team1Name;
    private String team2Name;
    private String startTime;
    private String score;
}

File: front/amadeusSports/src/app/dto/AddMatchDto.ts
export class AddMatchDto {
    team1Name!:string;
    team2Name!:string;
    startTime!: string;
    score!: String;
    constructor(team1Name:string, team2Name:string, startTime:string) {
        this.team1Name = team1Name;
        this.team2Name = team2Name;
        this.startTime = startTime;
        this.score = '-';
    }
}